packages/realm_dart/lib/src/handles/web/decimal128.dart

```diff
@@ -1,9 +1,29 @@
 // Copyright 2024 MongoDB, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import 'dart:typed_data';
+
+import 'package:decimal/decimal.dart';
+import 'package:rational/rational.dart';
+
+import 'package:realm_dart/src/handles/native/convert.dart';
```

Comment:
Should this depend on something in native?

Reply:
No. Fixed.